rpcs3/Emu/Cell/Modules/libmixer.cpp

```diff
@@ -6,6 +6,13 @@
 #include "cellAudio.h"
 #include "libmixer.h"
 
+#if __GNUC__ >= 6
+#include <math.h>
+#define FABS(X) std::fabs(X)
```

Comment:
If that works with VS and others sure, since I am not able to try I wanted to keep non GCC code the same as before.